src/plugins/intel_gna/src/optimizer/gna_pass_manager.hpp

```diff
@@ -208,6 +208,11 @@ DECL_PASS(MoveFakeQuantizeLayerIntoQuantParams);
 */
 DECL_PASS(TransposeWeightsFromNCHWToNHWC);
 
+/**
+ * @brief fuse FullyConnected and Eltwise layers - also in case there is 1-dim Reshape between them
```

Comment:
```suggestion
 * @brief fuse FullyConnected and Eltwise layers, also in case there is a Reshape between them having input with only one dimension > 1
```